include/effects/matrix/PatternWeather.h

```diff
@@ -271,9 +298,8 @@ class PatternWeather : public LEDStripEffect
             highToday   = KelvinToLocal(jsonDoc["main"]["temp_max"]);
             loToday     = KelvinToLocal(jsonDoc["main"]["temp_min"]);
 
-            String iconIndex = jsonDoc["weather"][0]["icon"];
-            iconToday = iconIndex.toInt();
-            debugI("Got today's temps: Now %d Lo %d, Hi %d, Icon %d", (int)temperature, (int)loToday, (int)highToday, iconToday);
+            iconToday = String(jsonDoc["weather"][0]["icon"]);

```

Comment:
Same comment.

Reply:
Same comment. I found the correct way to do the type coercion.